crates/holochain/src/conductor/kitsune_host_impl/query_region_set.rs

```diff
@@ -0,0 +1,66 @@
+use std::sync::Arc;
+
+use holochain_p2p::{dht::prelude::*, dht_arc::DhtArcSet};
+use holochain_sqlite::prelude::*;
+use kitsune_p2p_types::config::KitsuneP2pTuningParams;
+use rusqlite::named_params;
+
+use crate::conductor::error::ConductorResult;
+
+/// The network module needs info about various groupings ("regions") of ops
+pub async fn query_region_set(
+    db: DbWrite<DbKindAuthored>,
+    topology: Topology,
+    strat: &ArqStrat,
+    dht_arc_set: Arc<DhtArcSet>,
+    tuning_params: &KitsuneP2pTuningParams,
+) -> ConductorResult<RegionSetLtcs> {
+    let arq_set = ArqBoundsSet::from_dht_arc_set(&topology, &strat, &dht_arc_set)
+        .expect("arc is not quantizable (FIXME: only use quantized arcs)");
```

Comment:
This panic has never been encountered, so I guess it's safe to take the FIXME out and leave it as an expect, as risky as it may seem..

The ultimate solution would be either:
- have AgentInfo store quantized arcs only, or:
- use a more flexible `from_dht_arc_set` which rounds continuous Arcs to the nearest valid quantized Arq